_posts/2019-05-06-live-view-with-presence.md

```diff
@@ -346,14 +346,12 @@ Once we fetch this list, we iterate over it to collect a list of the individual
     phx_ref: "TNV4PzRfyhw="
     user_id: 1
   },
-  "2" => %{
-    metas: [
-      %{
-        email: "beini@email.com",
-        first_name: "Beini",
-        phx_ref: "ZZ30QuoI/8s="
-        user_id: 1
-      }
+  %{
```

Comment:
Updates this to match the structure of the data based on the other record in the list and what the transformation is doing.